client/src/components/ComboBox/findMatches.ts

```diff
@@ -0,0 +1,47 @@
+// Language-sensitive string comparison.
```

Comment:
This whole file is a copy-paste of what we used in Draftail in v4.0 and v4.1, and the collator / contains setup are heavily taken from the different projects linked to below.